eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/command/TeleportToRandomPlayerCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.eternalcode.core.feature.teleport.command;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.configuration.implementation.PluginConfiguration;
+import com.eternalcode.core.notice.NoticeService;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+@Route(name = "teleport-to-random-player", aliases = {"tprp"})
```

Comment:
nazwa komendy jest nietypowa w porównaniu do komend jakie mamy obecnie takich jak np. `tp <player>` `rtp` `tppos` `tphere` 
celowałbym raczej w coś w stylu `tp -randomplayer`

Reply:
Proponuję zmianę głównie dlatego, że nie ma w utarciu co to jest `tprp` (w odróżnieniu do `rtp`)

Reply:
i don't speak polish, please translate it to inglisz.

Reply:
Hi vLuckyyy!
Please do not arrange a circus
Best regards Rollczi twój fan

Reply:
After discussion with my friend @rchomczyk, I think to just be named `tprp` without an alias.

Reply:
Hi, Suggest an alternative that will resolve the problem.  https://github.com/EternalCodeTeam/EternalCore/pull/546#discussion_r1349532118
> [...] there is no common understanding of what `tprp` is [...]

Reply:
PS. I respect @rchomczyk's opinion, but it does not solve the problem

Reply:
I am not in favor of this proposition. I would suggest joining a Voice channel for a more in-depth and nuanced discussion. Personally, I find the appearance of the flag used in the command to be unsatisfactory.

Why you say:
> I propose the change mainly because there is no common understanding of what `tprp` is (as opposed to `rtp`).

/rtp - Random Teleport.
/tp - Teleport.
/tppos - Teleport position.
/back - Teleport to last position.
/tphere - Teleport here.
/up - Teleport up.

??? Why do you not say the same think in review of the commands above? 

Following this line of reasoning, all teleport commands should be done in this format:

/tp -randomPosition
/tp -position <coords>
/tp -lastPosition
/tp -here <player>
/tp -up